spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/File.scala

```diff
@@ -17,16 +17,70 @@
 package com.intel.analytics.bigdl.utils
 
 import java.io._
-import java.nio.file.{Files, Paths}
 
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.{FSDataInputStream, FSDataOutputStream, FileSystem, Path}
 import org.apache.hadoop.io.IOUtils
 
+import scala.collection.mutable.ArrayBuffer
+
+/**
+ * A uniform approach to get Input/Output stream in BigDL.
+ * @param fileName
+ */
+private[bigdl] class File(fileName: String) {
+  private val inputStreams: ArrayBuffer[InputStream] = new ArrayBuffer[InputStream]()
+  private var outputStream: OutputStream = null
```

Comment:
why do we need a declared variable ? I didn't see any usage inside the class  (just assigin a created output stream to it and close it), the create method below just return the outoutstream to external reference and we could close it there as you do it now